core-services/image-mirroring/openshift-kni/OWNERS

```diff
@@ -0,0 +1,22 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
```

Comment:
Maybe remove this line (not true for this folder)?

Reply:
yep, fixed